### PR TITLE
📃 fix(grub-theme): theme backup

### LINKED ISSUE #739: Fatal error when trying to run Grub Theme
After running Grub Theme option the 1st time everything is fine, after that i get this error.

fatal: destination path 'Top-5-Bootloader-Themes' already exists and is not an empty directory. 

please add a check if this folder already exists and if it does procede to the next steps of the prompt.

### PR DESCRIPTION
**Pull request by adamperkowski**
## Type of Change
- [x] Bug fix

## Description

Backs up the (if) existing theme directory before downloading. 

## Issues / other PRs related
Resolves #739 

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no errors/warnings/merge conflicts.

(📃 is a shell script btw)

### CODE REVIEW

File: core/tabs/applications-setup/grub-theme.sh [hunk 1]
```diff
@@ -4,8 +4,12 @@
 
 themeinstall(){
     mkdir -p "$HOME/.local/share"
-    cd "$HOME/.local/share" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
-    cd "$HOME/.local/share/Top-5-Bootloader-Themes"
+    cd "$HOME/.local/share"
+    if [ -d 'Top-5-Bootloader-Themes' ]; then
+        mv 'Top-5-Bootloader-Themes' 'Top-5-Bootloader-Themes.bak'
+    fi
```
Comment (nnyyxxxx) [inline core/tabs/applications-setup/grub-theme.sh:10]: this is going to lead to infinite backup creation

File: core/tabs/applications-setup/grub-theme.sh [hunk 2]
```diff
@@ -4,8 +4,12 @@
 
 themeinstall(){
     mkdir -p "$HOME/.local/share"
-    cd "$HOME/.local/share" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
-    cd "$HOME/.local/share/Top-5-Bootloader-Themes"
+    cd "$HOME/.local/share"
+    if [ -d 'Top-5-Bootloader-Themes' ]; then
+        mv 'Top-5-Bootloader-Themes' 'Top-5-Bootloader-Themes.bak'
```
Comment (nnyyxxxx) [inline core/tabs/applications-setup/grub-theme.sh:9]:
```suggestion
        rm -r 'Top-5-Bootloader-Themes'
```

Comment (nnyyxxxx) [reply core/tabs/applications-setup/grub-theme.sh:9]: nobody is going to be editing the grub themes anyways

Comment (adamperkowski) [reply core/tabs/applications-setup/grub-theme.sh:9]: yeah, true

**Review by nnyyxxxx — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**